### PR TITLE
tv-series are coming up for both trending and recommendation

### PR DESCRIPTION
**Pull request by Nitish814112**

### CODE REVIEW

File: src/components/Content/Card.jsx [hunk 1]
```diff
@@ -64,24 +64,20 @@ const Card = ({ id, name, bookmark, poster_path, title, release_date, first_air_
...
-            ) : (
-              first_air_date?.slice(0,4)
-            ) }
+            { release_date?release_date.slice(0,4):first_air_date.slice(0,4)}
         </span>
         </li>
         
         <li className="flex items-center">
           <RiFilmFill className="mr-1 text-white" />
-          <span>{media_type?.toUpperCase()}</span>
```
Comment (yashtibrewal) [inline src/components/Content/Card.jsx:77]: Why are we removing this?

File: src/constants.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 export const TMDB_BASE_URL = 'https://api.themoviedb.org/3/';
 export const BASE_IMAGE_URL = "https://image.tmdb.org/t/p/w500";
-export const BASE_LOCAL_URL = "https://entertainment-app-backend-k2wa.onrender.com/";
+export const BASE_LOCAL_URL = "http://localhost:4000/";
```
Comment (yashtibrewal) [inline src/constants.js:3]: Please push the remote link which we had.

**Review by yashtibrewal — CHANGES_REQUESTED** [review]

File: src/components/Content/Card.jsx (hunk 1 above)
Comment (Nitish814112) [reply src/components/Content/Card.jsx:77]: Capitalized words(media_type) was not looking nice in UI, that's why i have changed media_type only

File: src/constants.js (hunk 2 above)
Comment (Nitish814112) [reply src/constants.js]: done

**Pull request merged**